common/lib/xmodule/xmodule/js/fixtures/hls/XXXXXXXXT114-V015600_0_/XXXXXXXXT114-V015600_0_.m3u8

```diff
@@ -0,0 +1,11 @@
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:11
+#EXT-X-MEDIA-SEQUENCE:0
+#EXTINF:9.576244,
+XXXXXXXXT114-V015600_0_0.ts
+#EXTINF:8.842178,
+XXXXXXXXT114-V015600_0_1.ts
+#EXTINF:9.609611,
+XXXXXXXXT114-V015600_0_2.ts
+#EXT-X-ENDLIST
```

Comment:
@muzaffaryousaf Yep! They're static. If you want something with a more generic ID string let me know, but otherwise it's all set.